docs/src/main/asciidoc/resteasy-reactive.adoc

```diff
@@ -391,6 +391,11 @@ public class Endpoint {
 }
 ----
 
+[NOTE]
+====
+If you using the `quarkus-smallrye-openapi` extension, you must add the annotation `@BeanParam` to the `Parameters`` parameter, otherwise the OpenAPI document will not be generated correctly.
```

Comment:
@geoand I wonder if we shouldn't document the use of `@BeanParam` instead just above as a lot of people are using the OpenAPI extension.
Or adjust SmallRye OpenAPI but I'm not sure it would be able to handle this shortcut.

Reply:
Good point. It's probably easier to just mention `@BeanParam` instead of the shortcut version

Reply:
Maybe a good solution could be to add the annotation to the example and add a comment to that part that this is required to meet the JAX-RS spec so other libraries like OpenAPI can handle it correctly. Because I guess other libraries could also have struggle with the shortcut.

Reply:
@drmaniac looks good to me!

Make sure you amend/squash your changes into a single commit.